### DISCUSSION TITLE
12.3.13 not found

### DISCUSSION THREAD

**Discussion by SkylerLutz** [IgniteUI/igniteui-angular#14818] [Q&A] [answered]
```
14:00:27  npm ERR! 404 Not Found - GET https://packages.infragistics.com/npm/js-licensed/@infragistics/igniteui-angular/-/igniteui-angular-12.3.21.tgz
14:00:27  npm ERR! 404 
14:00:27  npm ERR! 404  '@infragistics/igniteui-angular@12.3.21' is not in the npm registry.
14:00:27  npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
```

did you remove 12.3.13? Can you put it back please? our ci is broken

**Comment by RivaIvanova** [comment, marked as answer]
Hi @SkylerLutz ,

I have been looking into your question and see that you have mentioned that version 12.3.13 is not found, however, in the console log you are targeting version 12.3.21.

What I can say is that both versions are present:
![igniteui-angular-packages](https://github.com/user-attachments/assets/4665b077-860f-46f9-b0cb-b73f4351cbb1)

I have tested this behavior on my side and both versions are installed successfully.
```cmd
npm install @infragistics/igniteui-angular@12.3.13
npm install @infragistics/igniteui-angular@12.3.21
```
Considering this, it seems that everything with the package management service works correctly and as expected.

I assume that it could be an issue with licensing. Please submit a support case here: https://account.infragistics.com/support-cases so that our support staff can investigate this matter further and determine if there are issues with your subscription.

Additionally, please do consider upgrading to a newer version of the library as 12.3.x has been out of support for some time now.
